src/middleware.ts

```diff
@@ -54,12 +60,14 @@ export function createMiddleware(
             res.setHeader(name, value)
           })
 
-          if (delay) {
-            setTimeout(() => res.send(body), delay)
-            return
-          }
+          const json = await mockedResponse.json()
+
+          // if (delay) {
```

Comment:
We can remove the `delay`. It's a part of the response resolver now so it will run on the MSW's side.